jscomp/test/js_json_test.js

```diff
@@ -647,23 +647,23 @@ catch (exn){
         }));
 }
 
-eq("File \"js_json_test.ml\", line 296, characters 12-19", JSON.stringify([
+eq("File \"js_json_test.ml\", line 296, characters 12-19", Js_json.stringifyAny([
```

Comment:
How about extending the tests to make sure exceptions are handled as expected?

Reply:
> How about extending the tests to make sure exceptions are handled as expected?

`stringify` throws the exception with the circular reference data and BigInt. I tried to make the circular reference data with `bs.raw` first, but it fails to run `./script/config` at the beginning. So, I add the test with `BigInt`.